pydata_sphinx_theme/docs-navbar.html

```diff
@@ -1,10 +1,17 @@
 
 <div class="container-xl">
 
+
     {% if logo %}
-    <a class="navbar-brand" href="{{ pathto('index') }}">
-      <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-    </a>
+      {% if theme_logo_doc %}
```

Comment:
Could we make this even simpler? The config could be `logo_link`, and we could do a quick check to see if it begins with `http`. If so, assume it's an external URL and insert it directly. If not, assume it's a doc in the website and use the `pathto(` function.

Reply:
Yeah, that sounds like a better idea indeed, more flexibility